packages/notifier/src/notifier.js

```diff
@@ -152,7 +153,106 @@ export const makeNotifierKit = (...initialStateArr) => {
 };
 
 /**
- * Adaptor from async iterable to notifier.
+ * @template T
+ * @param {ERef<Subscriber<T>>} subscriberP
+ * @returns {Promise<Notifier<T>>}
+ */
+export const makeNotifierFromSubscriber = async subscriberP => {
```

Comment:
```suggestion
export const makeNotifierFromSubscriber = async subscriber => {
```
I assume `P` is for "Promise" but that's not exactly what this is.

Reply:
It's an ERef, which _may_ be a promise. I think we do use this pattern to highlight that potential.

Reply:
I've also seen `P` for Presence, kind of the opposite.

I don't think we have any consistent guidelines on this but I would make the case that when possible we rely on the type system for whether something is a promise or not. If there's no need to bind both the promise and the resolved value then use the simpler name. non-blocking request.